notebooks/data-sources/TestGrid/metrics/persistent_failures_analysis.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
<p>Can you add the linked issues here? I think its #116. So adding the below to the markdown cell should work: </p><p> </p><p> </p><p>"[related issue #116](https://github.com/aicoe-aiops/ocp-ci-analysis/issues/116)"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>